orchestrator-salt/src/main/java/com/sequenceiq/cloudbreak/orchestrator/salt/states/SaltStates.java

```diff
@@ -145,7 +145,7 @@ public static ApplyFullResponse showState(SaltConnector sc, String state) {
     }
 
     private static Multimap<String, Map<String, String>> highStateJidInfo(SaltConnector sc, String jid) {
-        Map<String, List<Map<String, Object>>> jidInfo = sc.run("jobs.lookup_jid", RUNNER, Map.class, "jid", jid);
+        Map<String, List<Map<String, Object>>> jidInfo = sc.run("jobs.lookup_jid", RUNNER, Map.class, "jid", jid, "missing", "True");
```

Comment:
I don't quite understand what the purpose of `missing` or `True` are in this context. Is there existing code which checks for these values?

Reply:
this flag is passed to SaltStack: https://docs.saltproject.io/en/latest/ref/runners/all/salt.runners.jobs.html#salt.runners.jobs.lookup_jid
so `jid` paramater will take the value of the `jid` variable
`missing` parameter takes the value `True`
salt will include missing nodes in the response after this is set.